en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -230,6 +230,65 @@ When there are null values in custom expressions and are:
 * Fields without attributes, the null value is ignored. If the value in a required field, an error message occurs. If the value is in an optional field, the value is ignored and not sent to the connector. +
 * Fields with attributes, the value is ignored and not sent to the connector.
 
+=== Create a Picklist Mapping
+
+Using Picklist Mapping, you can map data from one application to the corresponding data needed by another application and then reuse those mappings in different flows in your organization. This allows you to connect two applications that have different values for similar fields and to align the values in those fields across applications. This enables you to determine how the possible values of an output field (data pill) map to the possible values of an input field.
```

Comment:
Confirm capitalization of Picklist Mapping--we don't typically capitalize feature names